WordPressKitTests/JetpackProxyServiceRemoteTests.swift

```diff
@@ -0,0 +1,122 @@
+import XCTest
+@testable import WordPressKit
+
+class JetpackProxyServiceRemoteTests: XCTestCase {
+    let timeout: TimeInterval = 1.0
+    let api = MockWordPressComRestApi()
```

Comment:
This would initialize a new `JetpackProxyServiceRemote` every accesss but the api would reference to the same object. Is this intended? Seems a bit error prone

Reply:
No, it should work as expected. The `MockWordPressComRestApi` instances are recreated because the test class is recreated for every test case run (as per [your comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/20031#discussion_r1093138369) 😆). Here's a console print:

```
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff4000>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff40e0>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff41c0>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff42a0>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff4380>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff4460>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff4540>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff4620>
>>> CALL: <WordPressKitTests.MockWordPressComRestApi: 0x600001ff4700>
```